internal/testrunner/runners/system/servicedeployer/compose.go

```diff
@@ -76,47 +77,17 @@ func (r *DockerComposeServiceDeployer) SetUp(inCtxt ServiceContext) (DeployedSer
 	serviceContainer := fmt.Sprintf("%s_%s_1", service.project, serviceName)
 	outCtxt.Hostname = serviceContainer
 
-	// Redirect service container's STDOUT and STDERR streams to files in local logs folder
-	localLogsFolder := inCtxt.Logs.Folder.Local
-	agentLogsFolder := inCtxt.Logs.Folder.Agent
-
-	service.stdoutFilePath = filepath.Join(localLogsFolder, stdoutFileName)
-	logger.Debugf("creating temp file %s to hold service container %s STDOUT", service.stdoutFilePath, serviceContainer)
-	outFile, err := os.Create(service.stdoutFilePath)
-	if err != nil {
-		return nil, errors.Wrap(err, "could not create STDOUT file")
-	}
-	service.stdout = outFile
-	outCtxt.STDOUT = agentLogsFolder + stdoutFileName
-
-	service.stderrFilePath = filepath.Join(localLogsFolder, stderrFileName)
-	logger.Debugf("creating temp file %s to hold service container %s STDERR", service.stderrFilePath, serviceContainer)
-	errFile, err := os.Create(service.stderrFilePath)
-	if err != nil {
-		return nil, errors.Wrap(err, "could not create STDERR file")
-	}
-	service.stderr = errFile
-	outCtxt.STDERR = agentLogsFolder + stderrFileName
-
-	logger.Debugf("redirecting service container %s STDOUT and STDERR to temp files", serviceContainer)
-	cmd := exec.Command("docker", "attach", "--no-stdin", serviceContainer)
-	cmd.Stdout = service.stdout
-	cmd.Stderr = service.stderr
-
-	if err := cmd.Start(); err != nil {
-		return nil, errors.Wrap(err, "could not redirect service container STDOUT and STDERR streams")
-	}
-
+	// Connect service network with stack network (for the purpose of metrics collection)
 	stackNetwork := fmt.Sprintf("%s_default", stack.DockerComposeProjectName)
-	logger.Debugf("attaching service container %s to stack network %s", serviceContainer, stackNetwork)
```

Comment:
Maybe bring back the debug logs so we can tell what's going on while troubleshooting?

Reply:
Fixed, thanks!